src/types/xdai/xdai.ts

```diff
@@ -8,43 +8,10 @@ import os from 'os';
 import path from 'path';
 import fs from 'fs-extra';
 import { filterVersionsByNetworkType } from '../../util';
+import {openEthereumConfig} from './openethereum/config';
```

Comment:
When destructuring, please add a single space on each side within the brackets like you see in the import immediately above.